File: dataset/mruby/fuzz/in/control.rb
# Control flow
5.times do |i|
  puts i
end

x = 10
while x > 0
  x -= 1
end

if x == 0
  puts "done"
else
  puts "error"
end

case x
when 0
  puts "zero"
else
  puts "other"
end

File: dataset/duktape/fuzz/in/control.js
// Control flow
for (var i = 0; i < 5; i++) {
    print(i);
}

var x = 10;
while (x > 0) {
    x--;
}

if (x === 0) {
    print("done");
} else {
    print("error");
}

switch (x) {
    case 0:
        print("zero");
        break;
    default:
        print("other");
}
